PO.md

```diff
@@ -0,0 +1,54 @@
+Obowiązkowa definicja:​ ​https://craftware.pl/kim-nie-jest-product-owner/ definicja wizji:
+Wizja to nie tylko akt twórczy ale przede wszystkim rzetelny proces analityczny oparty o potrzeby rynku i sponsorów, który jest spisany w postaci Roadmapy produktu opartym o USER STORIES.
+### CEL
+Definiowanie opłacalnych i regularnych przyrostów biznesowych, które mają mierzalną korzyść dla użytkowników produktu i pozwalają iteracyjnie rozwijać produkt / usługę, zgodnie z potrzebami rynku, i w zgodzie z przyznanym budżetem.
+### KPI
+1. Jaki % sprintów został wydany skutecznie na produkcję
+2. Przyjmowalność sprintów przez Stakeholderów i sponsorów
+3. PO Przychodzi przygotowany na planowanie (ma gotowe KA na sprint)
+4. Czytelność backlogu i Kryteria Akceptacji User Stories dla zespołu DEV
+5. Pilnowanie budżetu projektu i ROI z inwestycji
+6. Zdany​ ​scrum.org​ Product Owner Open Assessment (junior) lub uzyskanie
+certyfikatu Proffesional Scrum Product Owner (mid / senior) przed rozpoczęciem projektu.
+
+### Główne Narzędzie pracy PO
+1. Jira
+2. Planner
+3. Cost Tracker
+   
+### Obowiązki Product Ownera
+* Zarządzanie Backlogiem Produktu
+* Opracowywanie i komunikacje Celu Produktu
+* Negocjacje z zespołem zakresy sprintów i odbiera ich realizację;
+* Publiczne opowiadanie o projekcie / produkcie, występowanie w materiałach
+promocyjnych
+* Maksymalizacja wartości pracy będącej efektem działań Scrum Teamu;
+* Monitorowanie budżetu i końcowy wynik produktu;
+* Planowanie Kamieni Milowych oraz ich realizacje wedle ustalonych z
+Klientem terminów;
+* Odpowiada za jasne zakomunikowanie celu projektu oraz elementów
+Backlogu Projektu Zespołowi Scrumowemu;
+* Odpowiada za komunikację z Interesariuszami i sponsorami w obszarze
+postępu prac oraz timeline wydań
+* Zarządza ryzykami oraz ich mitygacją na poziomie produktu;
+* Tworzy Elementy Backlogu Produktu i Zarządza Backlogiem Produktu przez :
+* Określanie potrzeb biznesowych w formie User Story zgodnie z modelem ​I.N.V.E.S.T​ w oparciu o całą posiadaną wiedzę na temat produktu, który powinien być aktualizowany wraz ze pojawiającymi się nowymi potrzebami biznesowymi;
+* Opracowywanie propozycji kryterium akceptacji dla każdego User Story w Backlogu Produktu, które może być doprecyzowywane w trakcie Refinementu lub Planowania;
+* Odpowiada za priorytetyzację Backlogu Produktu;
+* Jest odpowiedzialny za zbieranie i filtrowanie potrzeb interesariuszy
+oraz decyzję o ich uwzględnieniu w Backlogu w celu maksymalizacji Wartości Produktu.
+Obowiązki w obszarze Kontrolingu
+* Planowanie, aktualizowanie zapotrzebowania na członków zespołu w Tempo Planner;
+* Analizę i planowanie zespołu Tempo Planner w oparciu o informacje dostępne w Cost Tracker
+  design by​ Laboratorium EE
+* Przygotowywania na koniec Sprintu raportu dotyczącego stopnia realizacji budżetu z perspektywy ostatniego wydanego Inkrementu;
+Obowiązki Dotyczące Narzędzi
```

Comment:
To nie jest osobnym punktem, więc wyświetla się przyklejone do poprzedniego punktu. Prawdopodobnie to powinien być punkt, a kolejne punkty powinny być jego podpunktami (czyli mieć dodatkowe wcięcie przed gwiazdką).